horovod/mxnet/mpi_ops.py

```diff
@@ -166,6 +176,8 @@ def broadcast(tensor, root_rank, name=None):
         tensor: A tensor to broadcast.
         root_rank: The rank to broadcast the value from.
         name: A name of the broadcast operation.
+        priority: The priority of this operation. Higher priority operations are
+                  likely to be executed before other actions.
```

Comment:
nit: change the word "actions" to "operations" to be consistent

Reply:
Done